engine/src/test/java/io/camunda/zeebe/engine/metrics/JobMetricsTest.java

```diff
@@ -96,7 +96,7 @@ public void shouldCountTimedOut() {
     ENGINE.jobs().withType(JOB_TYPE).withTimeout(timeout.toMillis()).activate();
 
     // when
-    ENGINE.getClock().addTime(timeout);
+    ENGINE.getClock().addTime(timeout.plus(Duration.ofMillis(50)));
```

Comment:
:wrench: We could make the time shift less fuzzy by using the record timestamp instead.

The additional time of 50 milliseconds might be enough in most cases. But in the CI, it could take longer. 

In the implementation, it adds the timeout to the timestamp of the `activate` command. We could do something similar by calculating the time shift based on the timestamp of the `activated` event that is returned by the activation.

Reply:
Clever! I like that 🍪